howto/select_observations.md

```diff
@@ -111,9 +111,9 @@ python examples/observation_space.py env=atari agent=dreamer_v3 env.id=MsPacmanN
 
 > **Note**
 >
-> You can try to override some *cnn* or *mlp* keys by specifying the `cnn_keys.encoder` and the `mlp_keys.encoder` arguments. **Not all** environments allow it.
+> You can try to override some *cnn* or *mlp* keys by specifying the `algo.cnn_keys.encoder` and the `algo.mlp_keys.encoder` arguments. **Not all** environments allow it.
 > 
-> For instance, the `python examples/observation_space.py env=gym agent=dreamer_v3 env.id=LunarLander-v2 cnn_keys.encoder=[custom_cnn_key] mlp_keys.encoder=[custom_mlp_key]` command will return the following observation space: 
```

Comment:
With the changes in the configs and `sheeprl.utils.envs.make_env()` function, this script no longer works. Do we keep (and modify) it or delete it?